### PR TITLE
Add an enum for the `forceScreen` variable

### PR DESCRIPTION
**Pull request by smcmurtry**
# Summary | Résumé

This is in response to a bug in #1316 where we checked that no screen was forced with:
```
if (forceScreen) {
// assume that user is forcing a screen
}
```

This was a bug because `forceScreen === 'None'` when the user makes a selection and then resets the option. Using an enum will reduce the chances we will make this mistake again.

### CODE REVIEW

File: src/screens/home/HomeScreen.tsx [hunk 1]
```diff
@@ -10,16 +10,16 @@ import {
...
 } from 'services/ExposureNotificationService';
 import {useStorage} from 'services/StorageService';
+import {RegionCase} from 'shared/Region';
 import {getRegionCase} from 'shared/RegionLogic';
 import {usePrevious} from 'shared/usePrevious';
+import {ForceScreen} from 'shared/ForceScreen';
 import {useRegionalI18n} from 'locale';
 
-import {useExposureNotificationSystemStatusAutomaticUpdater} from '../../services/ExposureNotificationService';
-import {RegionCase} from '../../shared/Region';
```
Comment (smcmurtry) [inline src/screens/home/HomeScreen.tsx:21]: This is an unrelated change - just thought I'd clean up the imports a bit

**Review by timarney — APPROVED** [review]

**Comment by timarney** [comment]
🎉
